project_task_material_stock/models/stock_move.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Valentin Vinagre <valentin.vinagre@qubiq.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    task_material_id = fields.One2many(
+        "project.task.material", "stock_move_id", string="Project Task Material",
+    )
+
+    def _action_done(self):
```

Comment:
```suggestion
    def _action_done(self, cancel_backorder=False):
```

Method header changed in this version.